server/src/services/menuItemServices.ts

```diff
@@ -7,7 +7,7 @@ export class MenuItemService {
   static async addMenuItem(data: any) {
     try {
       const newMenuItem = await fullMenuItemPresets.create(data);
-      cachedMenuItems.invalidate("all");
+      cachedMenuItems.invalidateAll();
```

Comment:
why invalidate the entire cache and not just 'all'? 'all' is for the getting all menu items so if you add a new one theres no reason to invalidate the getOneMenuItem caches